app/helpers/reports_helper.rb

```diff
@@ -1,6 +1,10 @@
 module ReportsHelper
   def year_only(report)
```

Comment:
Predicate methods (methods that return `true` or `false` are normally suffixed with `?`, i.e. `year_only?`.